test/cbmc/proofs/ARP/ARPGenerateRequestPacket/ARPGenerateRequestPacket_harness.c

```diff
@@ -24,9 +28,15 @@ void harness()
     NetworkBufferDescriptor_t xNetworkBuffer2;
 
     xNetworkBuffer2.pucEthernetBuffer = xBuffer;
-    xNetworkBuffer2.xDataLength = ucBUFFER_SIZE;
 
-    /* vARPGenerateRequestPacket asserts buffer has room for a packet */
+    xNetworkBuffer2.xDataLength = ucBUFFER_SIZE;
+    /* vARPGenerateRequestPacket asserts buffer has room for a packet. */
     __CPROVER_assume( xNetworkBuffer2.xDataLength >= sizeof( ARPPacket_t ) );
+
+    /* Non-deterministically allocate some memory or return a NULL. */
+    xNetworkBuffer2.pxEndPoint = safeMalloc( sizeof( *xNetworkBuffer2.pxEndPoint ) );
```

Comment:
You use malloc above and safeMalloc here, I think you should choose one or the other.

Reply:
I choose safeMalloc :)